resources/views/dashboard/admin/events/edit.blade.php

```diff
@@ -56,8 +56,10 @@
             {!! Form::textArea('description', $event->description, ['class' => 'form-control text-editor']) !!}
         </div>
         <div class="form-group">
-            {!! Form::label('banner', 'Upload Banner') !!}
-            {!! Form::file('banner', ['class' => 'form-control']) !!}
+            {!! Form::label('banner', 'Upload Banner or Enter Banner URL (Optional)') !!} <!-- Adjusted label -->
+            {!! Form::file('banner', ['class' => 'form-control']) !!} <!-- Adjusted file input -->
+            <Text>OR</Text>
```

Comment:
Where did you get this from? `<Text>` is not an HTML element as far as I can tell.... use a `<span>` or `<p>` instead.

Reply:
Yeah I have no idea where that came from I think I picked up the habit from Flutter.